modules/preview-react/radio/stories/radio.stories.mdx

```diff
@@ -0,0 +1,133 @@
+import {Meta} from '@storybook/addon-docs';
+import {Specifications, SymbolDoc} from '@workday/canvas-kit-docs';
+
+import {RadioGroup2} from '@workday/canvas-kit-preview-react/radio';
+
+import {Basic} from './examples/Basic';
+import {Alert} from './examples/Alert';
+import {Error} from './examples/Error';
+import {Inverse} from './examples/Inverse';
+import {LabelPosition} from './examples/LabelPosition';
+import {NoValue} from './examples/NoValue';
+import {RefForwarding} from './examples/RefForwarding';
+import {Disabled} from './examples/Disabled';
+import {Required} from './examples/Required';
+
+<Meta title="Preview/Radio" component={RadioGroup2} />
```

Comment:
Removing the "2" here as well. Also, it seems like this should go in `Preview > Inputs` in SB along with the Preview TextArea and TextInput.

```suggestion
<Meta title="Preview/Inputs/Radio" component={RadioGroup} />
```